content/en/docs/collector/internal-telemetry.md

```diff
@@ -0,0 +1,163 @@
+
+```yaml
+service:
+  telemetry:
+    metrics:
+      level: detailed
+      address: ':8888'
+```
+
+The Collector can also be configured to scrape its own metrics and send them
+through configured pipelines. For example:
+
+```yaml
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'otelcol'
+          scrape_interval: 10s
+          static_configs:
+            - targets: ['0.0.0.0:8888']
+          metric_relabel_configs:
+            - source_labels: [__name__]
+              regex: '.*grpc_io.*'
+              action: drop
+exporters:
+  debug:
+service:
+  pipelines:
+    metrics:
+      receivers: [prometheus]
+      processors: []
+      exporters: [debug]
+```
+
+### Configuring logs
+
+You can find log output in `stdout`. The verbosity level for logs defaults to
+`INFO`, but you can adjust it in the config `service::telemetry::logs`:
+
+```yaml
+service:
+  telemetry:
+    logs:
+      level: 'debug'
+```
+
+You can also see logs for the Collector on a Linux systemd system using
+`journalctl`:
+
+{{< tabpane text=true >}} {{% tab "All logs" %}}
+
+```sh
+journalctl | grep otelcol
+```
+
+{{% /tab %}} {{% tab "Errors only" %}}
+
+```sh
+journalctl | grep otelcol | grep Error
+```
+
+{{% /tab %}} {{< /tabpane >}}
+
+### Configuring traces
+
+Although the Collector does not expose traces by default, an effort is underway
+to
+[change this](https://github.com/open-telemetry/opentelemetry-collector/issues/7532).
+The work includes supporting configuration of the OpenTelemetry SDK used to
+produce the Collector's internal telemetry. This feature is currently behind two
+feature gates:
+
+```sh
+--feature-gates=telemetry.useOtelWithSDKConfigurationForInternalTelemetry
+```
+
+The gate `useOtelWithSDKConfigurationForInternalTelemetry` enables the Collector
+to parse configuration that aligns with the
+[OpenTelemetry Configuration](https://github.com/open-telemetry/opentelemetry-configuration)
+schema. Support for this schema is still experimental, but it does allow
+telemetry to be exported using OTLP.
+
+The following configuration can be used in combination with the aforementioned
+feature gates to emit internal metrics and traces from the Collector to an OTLP
+backend:
+
+```yaml
+service:
+  telemetry:
+    metrics:
+      readers:
+        - periodic:
+            interval: 5000
+            exporter:
+              otlp:
+                protocol: grpc/protobuf
+                endpoint: https://backend:4317
+    traces:
+      processors:
+        - batch:
+            exporter:
+              otlp:
+                protocol: grpc/protobuf
+                endpoint: https://backend2:4317
+```
+
+See the
+[example configuration](https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/kitchen-sink.yaml)
+for additional configuration options.
+
+> Note that this configuration does not support emitting logs as there is no
+> support for logs in OpenTelemetry Go SDK at this time.
```

Comment:
Is this still true? @codeboten added this note in November, but the [issue](https://github.com/open-telemetry/opentelemetry-go/issues/3827) linked in the [repo doc](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/observability.md#how-we-expose-telemetry) was closed on February 26.

Reply:
maybe skip that note?

if we keep it use the following formatting:

```suggestion
{{% alert title="Note" %}}

Note that this configuration does not support emitting logs as there is no
support for logs in OpenTelemetry Go SDK at this time.

{{% /alert %}}
```